nemo/collections/nlp/data/datasets/sgd_dataset/data_processor.py

```diff
@@ -35,11 +36,13 @@
 __all__ = ['FILE_RANGES', 'PER_FRAME_OUTPUT_FILENAME', 'SGDDataProcessor']
 
 FILE_RANGES = {
-    "sgd_single_domain": {"train": range(1, 44), "dev": range(1, 8), "test": range(1, 12)},
-    "sgd_multi_domain": {"train": range(44, 128), "dev": range(8, 21), "test": range(12, 35)},
-    "sgd_all": {"train": range(1, 128), "dev": range(1, 21), "test": range(1, 35)},
+    "dstc8_single_domain": {"train": range(1, 44), "dev": range(1, 8), "test": range(1, 12)},
+    "dstc8_multi_domain": {"train": range(44, 128), "dev": range(8, 21), "test": range(12, 35)},
+    "dstc8_all": {"train": range(1, 128), "dev": range(1, 21), "test": range(1, 35)},
     "multiwoz": {"train": range(1, 18), "dev": range(1, 3), "test": range(1, 3)},
     "debug_sample": {"train": range(1, 2), "dev": range(1, 2), "test": range(1, 2)},
+    "sgdplus_single": {"train": range(1, 2), "dev": range(1, 2), "test": range(1, 2)},
+    "sgdplus_all": {"train": range(1, 3), "dev": range(1, 3), "test": range(1, 3)},
```

Comment:
should these be `dstc8_plus` for consistency?

Reply:
The way I was thinking was sgdplus is the name of the new dataset. That dstc8 refers to the split used in dstc8 mostly because of those prefixes of "all" and "single_domain". SGD refers to the dataset itself. So we created another split from SGD named sgdplus. Does it look OK?